src/features/user/MFV2/InfoAndCTA/DonorCircleInvitation/index.tsx

```diff
@@ -1,10 +1,54 @@
 import React from 'react';
 import styles from '../InfoAndCta.module.scss';
+import DonorCircleInvitationHeader from './DonorCircleInvitationHeader';
+import SingleInvitationCard from './SingleInvitationCard';
+import { useTranslations } from 'next-intl';
 
 const DonorCircleInvitation = () => {
+  const t = useTranslations('Profile');
+
+  const donorCircleCardsData = [
+    {
+      link: 'https://www.plant-for-the-planet.org/children-youth/',
+      image: '/assets/images/donorCircleInvitationImages/youthEmpowerment.png',
+      title: t('infoAndCtaContainer.youthEmpowermentTitle'),
+      subtext: t('infoAndCtaContainer.youthEmpowermentSubtext'),
+    },
+    {
+      link: 'https://www.plant-for-the-planet.org/restoration-organizations/',
+      image: '/assets/images/donorCircleInvitationImages/restorationTools.png',
+      title: t('infoAndCtaContainer.restorationToolsTitle'),
+      subtext: t('infoAndCtaContainer.restorationToolsSubtext'),
+    },
+    {
+      link: 'https://www.plant-for-the-planet.org/advice/',
+      image: '/assets/images/donorCircleInvitationImages/restorationAdvice.png',
+      title: t('infoAndCtaContainer.restorationAdviceTitle'),
+      subtext: t('infoAndCtaContainer.restorationAdviceSubtext'),
+    },
+    {
+      link: 'https://plant-for-the-planet.org/yucatan/',
+      image:
+        '/assets/images/donorCircleInvitationImages/forestRestorationAndConservation.png',
+      title: t('infoAndCtaContainer.restorationAndConservationTitle'),
+      subtext: t('infoAndCtaContainer.restorationAndConservationSubtext'),
+    },
+  ];
+
   return (
     <div className={styles.donorCircleInvitationContainer}>
-      <h2 className={styles.headerTitle}>DonorCircleInvitationContainer</h2>
+      <DonorCircleInvitationHeader />
+      <div className={styles.inviationCardsContainer}>
+        {donorCircleCardsData.map((item, index) => (
+          <SingleInvitationCard
+            image={item.image}
+            title={item.title}
+            subtext={item.subtext}
+            link={item.link}
+            key={index}
+          />
+        ))}
+      </div>
```

Comment:
UI wise, the invitationCardsContainer needs more space between the items. Please ref Figma design and make it uniform. I think there is some padding around the card contents in Figma designs, so the space appears more.